From f6c173b1c32e6dcbac510a4bcb60e6bb080c146a Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Wed, 31 May 2017 12:26:04 +0500
Subject: [PATCH 2/2] spirv-lesspipe.sh: allow using generic shells

The script is harmless for any type of shell and
shouldn't be tied with bash to allow catering
more possibilities.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 tools/lesspipe/spirv-lesspipe.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/lesspipe/spirv-lesspipe.sh b/tools/lesspipe/spirv-lesspipe.sh
index 81e3355..f955259 100644
--- a/tools/lesspipe/spirv-lesspipe.sh
+++ b/tools/lesspipe/spirv-lesspipe.sh
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 # Copyright (c) 2016 The Khronos Group Inc.
 
 # Licensed under the Apache License, Version 2.0 (the "License");
-- 
2.11.1

